back/scripts/utils/dataframe_operation.py

```diff
@@ -13,13 +14,17 @@
 
 
 # Function to merge duplicate columns in a DataFrame
-def merge_duplicate_columns(df):
+def merge_duplicate_columns(df: pd.DataFrame, separator: str = " / ") -> pd.DataFrame:
     df.columns = df.columns.astype(str)
```

Comment:
```suggestion
    df = df.rename(columns=str)
```
La onction rename accepte une fonction et ca permet de retourner un dataset entier plutot que modifier en place ses properties.

Reply:
Merci pour la notation, élégant en effet.
Mais concrètement, cette ligne est retirée car hors scope et dangereuse : on modifie le nom des colonnes sans prévenir et sans réel besoin. Cette méthode fonctionne très bien avec des noms de colonne d'un autre type. Mieux vaut faire explicitement la manipulation avant son appel plutôt que discrètement dedans.